src/github/queries.js

```diff
@@ -17,6 +17,79 @@ const me = () => {
   });
 };
 
+const getContributorsData = (data, info) => {
+  const inputs = `
+    $orgID: ID!, 
+    $login: String!, 
+    $firstRepos: Int!, 
+    $firstUsers: Int!, 
+    $afterRepos: String,
+    $afterUsers: String,
+  `;
+
+  const { id: orgID, login } = info.organization;
+  const { after: afterRepos, first: firstRepos } = info.repoArgs;
+  const { after: afterUsers, first: firstUsers } = info.userArgs;
+
+  const variables = {orgID, login, firstRepos, firstUsers, afterRepos, afterUsers};
+
+  const query = `
+    query getFirstContributionDate(${inputs}) {
+      organization(login: $login) {
+        repositories(first: $firstRepos after: $afterRepos) {
+          nodes {
+            mentionableUsers(first: $firstUsers after: $afterUsers) {
+              nodes {
+                ${data}
+              }
+            }
+          }
+        }
+      }
+    }
+  `;
+
+  return JSON.stringify({query, variables});
+}
+
+/**
+ * Fetch contributors from the specified organization
+ * @param {object} organization
+ * @param {object} repositoryArgs object with args (first, after, etc) to filter repositories
+ * @param {object} userArgs object with args (first, after, etc) to filter metionableUsers
+ */
+const contributors = (organization, repoArgs, userArgs) => {
+  const data = `
+    id
+    name
+    bio
+    location
+    email
+    login
+    websiteUrl
```

Comment:
This part is pretty similar with [this](https://github.com/panelinhadees/server/blob/7fbecb10de0b391d423137d793a9c70014e05720/src/github/queries.js#L5-L11) I think in use a`Fragment`